dumont/roster.json

```diff
@@ -10,6 +10,9 @@
       ]
     },
     "students": [
-      
+      {
+        "name": "Brandon",
+        "github": "Branleaf"
+      }
```

Comment:
Hi Brandon,
Can you please add your surname as well :) ?

Reply:
sure thing, should be there now